lib/lib_rss.php

```diff
@@ -856,7 +856,8 @@ function checkTrustedIP(): bool {
 }
 
 function httpAuthUser(bool $onlyTrusted = true): string {
-	$auths = array_intersect_key($_SERVER, ['REMOTE_USER' => '', 'REDIRECT_REMOTE_USER' => '', 'HTTP_REMOTE_USER' => '', 'HTTP_X_WEBAUTH_USER' => '']);
+	$check_array = array_intersect_key($_SERVER, array_flip(['REMOTE_USER', 'REDIRECT_REMOTE_USER', 'HTTP_REMOTE_USER', 'HTTP_X_WEBAUTH_USER']));
+	$auths = array_unique($check_array);
```

Comment:
```suggestion
	$auths = array_intersect_key($_SERVER, ['REMOTE_USER' => '', 'REDIRECT_REMOTE_USER' => '', 'HTTP_REMOTE_USER' => '', 'HTTP_X_WEBAUTH_USER' => '']);
	$auths = array_unique($auths);
```

If possible, I would prefer the previous version than with `array_flip()`, to avoid a function call and arguably clearer